packages/superset-ui-plugin-chart-table/src/processMetrics.ts

```diff
@@ -20,3 +23,10 @@ export default function processMetrics({
     .concat(processedPercentMetrics)
     .filter(m => typeof records[0][m] === 'number');
 }
+
+export default createSelector(
```

Comment:
You better export a factory `() => createSelector(...)` so it can be used multiple times.